endpoints/openrtb2/auction.go

```diff
@@ -1356,10 +1356,15 @@ func (deps *endpointDeps) processStoredRequests(ctx context.Context, requestJson
 		return nil, nil, errs
 	}
 
+	bidRequestID, err := getBidRequestID(requestJson)
+	if err != nil {
+		return nil, nil, []error{err}
+	}
+
 	// Apply the Stored BidRequest, if it exists
 	resolvedRequest := requestJson
 
-	if deps.cfg.GenerateRequestID {
+	if deps.cfg.GenerateRequestID || bidRequestID == "{{UUID}}" {
```

Comment:
Does this work for AMP requests too?

Reply:
This doesn't work for AMP, so I'll update this PR so that it does!